### DISCUSSION TITLE
With windows 10 file explorer, when i click on a tab and click on home, explorer crashes.

### DISCUSSION THREAD

**Discussion by SystemEdmond** [valinet/ExplorerPatcher#3217] [Q&A]
title explains

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

then try this:

```
vivetool /reset /id:37634385

reboot

vivetool /disable /id:37634385

vivetool /enable /id:37634385

reboot
```

this is not a bug with EP
